src/ToolbarAction.js

```diff
@@ -9,7 +9,7 @@ L.ToolbarAction = L.Handler.extend({
 			className: '',
 			tooltip: ''
 		},
-		subToolbar: new L.Toolbar()
+		subToolbar: L.toolbar()
```

Comment:
Let's keep this as `new L.Toolbar()`, for consistency with Leaflet. Leaflet exposes factory functions for external use, but tends to use constructors internally. See: https://github.com/Leaflet/Leaflet/blob/master/src/layer/marker/Marker.js#L10